ajax/libs/Leaflet.EasyButton/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Leaflet.EasyButton",
+  "filename": "easy-button.js",
```

Comment:
@davidtgq `filename` should point to the minified main file, so let's use `"easy-button.min.js"`. Thanks.